db/src/db.ts

```diff
@@ -5,13 +5,16 @@ import { compare } from 'fast-json-patch';
 import LevelUpCtor, { LevelUp } from 'levelup';
 import LevelDown from 'leveldown';
 import { Mutex } from 'async-mutex';
+import { Version } from '@binaris/shift-interfaces/dist/db';
 import {
   UpdateOptions,
-  Version,
   Versioned,
-  Patch,
   KeyedPatches,
 } from '@binaris/shift-interfaces/dist/subscriptions';
+import {
+  Patch,
+  Serializable,
+} from '@binaris/shift-interfaces-node-client/dist/interfaces';
```

Comment:
I understand why you did this but I don't like it.
I'd have preferred we have a pre-rush step to generate the required code: `npx concord@<version> --nocompile ..`.
I also understand that you're thinking of changing concord to fit into this situation, I thought about it a bit and it sounds like a legit option although I would still like to avoid it.

Reply:
Talking to @vogre I was convinced that modifying concord to not create a full package is a good option and should work well for us.